<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>963</width>
    <height>782</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>50</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>963</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>采集</string>
    </property>
    <addaction name="actionconfig"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>显示</string>
    </property>
   </widget>
   <widget class="QMenu" name="viewMenu">
    <property name="title">
     <string>视图</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>存储</string>
    </property>
    <addaction name="saveDataConfig"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="openPrjtaction"/>
    <addaction name="action"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>报告</string>
    </property>
    <addaction name="action_noReport"/>
    <addaction name="action_staticReport"/>
    <addaction name="action_DReport"/>
   </widget>
   <addaction name="menu_3"/>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="viewMenu"/>
   <addaction name="menu_4"/>
   <addaction name="menu_5"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionconfig">
   <property name="text">
    <string>config</string>
   </property>
   <property name="toolTip">
    <string>config</string>
   </property>
  </action>
  <action name="openPrjtaction">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>另存为</string>
   </property>
  </action>
  <action name="saveDataConfig">
   <property name="text">
    <string>存储设置</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>全色辅助数据</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>多光谱辅助数据</string>
   </property>
  </action>
  <action name="action_noReport">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>不生成报告</string>
   </property>
  </action>
  <action name="action_staticReport">
   <property name="text">
    <string>静态报告</string>
   </property>
  </action>
  <action name="action_DReport">
   <property name="text">
    <string>动态报告</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../Resource.qrc"/>
 </resources>
 <connections/>
</ui>
